data_store/cli.py

```diff
@@ -43,6 +44,18 @@ def seed_reference():
 
         print("Reference data seeded successfully.")
 
+    @database_cli.command("seed-sample-data")
```

Comment:
Nice! Do you mind just updating the README to include this as well as it has a list of all the CLI commands that you can run?

Reply:
Added! :+1: